versions.json

```diff
@@ -222,6 +222,14 @@
         "arm64v8",
         "ppc64le",
         "s390x"
+      ],
+	  "alpine3.13": [
+        "amd64",
+        "arm32v6",
+        "arm32v7",
+        "arm64v8",
+        "ppc64le",
+        "s390x"
```

Comment:
@nschonni like this?

Reply:
I am not nschonni but here's my 2 cts. Pls use spaces instead of tabs. The _eclint_ checker had reported: 
`202:01 ❌  invalid indent style: tab, expected: space`

Reply:
This should be fixed